### ISSUE TITLE
Show 'Unreliable signature' label when email was signed with a expired, revoked or wrong key

### ISSUE THREAD

**Issue by thaissiqueira** [pixelated/pixelated-user-agent#734]
When some email is signed with a expired, revoked or wrong key, Pixelated is showing 'Not signed' label.
Instead of that, we need to show the 'Unreliable signature' label.

Things to do:
- [x] Remove unused scss
- [x] Check translation keys

**Comment by thaissiqueira** [comment]
We are having problems to provision the vagrant box of pixelated-user-agent. We are working to solve it.

**Comment by thaissiqueira** [comment]
The vagrant problem is being addressed for #751.
We are configuring our own machine to run pixelated-user-agent to finish this issue.

**Comment by thaissiqueira** [comment]
Acceptance criteria to QA:

- [ ] When email has a valid signature, the label should be 'Verified sender' and green;
- [ ] When email hasn't a signature, the label should be 'Not signed' and yellow;
- [ ] When email has an invalid signature, the label should be 'Unreliable signature' and red.

These changes are in branch https://github.com/pixelated/pixelated-user-agent/commits/label-sign
Please, merge it after validate.

**Comment by cz8s** [comment]
I sent a unsigned email and that was shown as 'unreliable signature'

**Comment by thaissiqueira** [comment]
Yes, it is happening when email is encrypted and not signed. I'll take a look on that.

**Comment by thaissiqueira** [comment]
Approach changed!

Instead of show 'Not signed' when email hasn't a signature and 'Unreliable signature' when email has an invalid signature, we decide to just show 'Verified sender' when email has a valid signature and show nothing when the status of signature is different of signed.

We decided to follow this approach because:
- To show the correct label in all cases we have to PR big codes to LEAP, and we are not willing to do that at the moment.
- This approach is closer to @SweetVirginia  plan to avoid disturbing users with scare alerts.

So now, the acceptance criteria to QA is:

- [ ] When email has a valid signature, the label should be 'Verified sender' and green;
- [ ] When signature status is different of valid (for example hasn't a signature or has an invalid signature) we will show nothing about signature.

PS: This approach was used in other moment at list of sent email, when the status of encryption and signature was not shown.

These changes are in branch https://github.com/pixelated/pixelated-user-agent/commits/label-sign
Please, merge it on master after validate.

**Comment by cz8s** [comment]
I don't know if this is a good idea.
If I send a mail to you and pretend to be renata, the ui would not show there is a problem.
The only difference would be that the green 'Verified Sender' is not shown. 
Will users realize there is something strange happening just because a green button is missing?
Is this a scenario we should be aware of?
Would it be better to always show red 'Not signed' if something is wrong with the signature?

**Comment by cz8s** [comment]
Leap should have the same problem, right?
Maybe send a mail to leap and ask how they think this should be handled?

**Comment by SweetVirginia** [comment]
About signed emails: if you ask to any average user, specially the ones who are going to use Pixelated in a short term, they have no idea what a signature is. They are aware of cryptography (thanks WhatsApp), but signature is an abstraction they have no idea where it comes from. 
That is one of the reasons why I think we should test if verified sender status is better comprehended if next to sender/receiver's (to/from) name/email  and not at the message's top, as it is for encrypted. Once is closer to contact, is easier to relate that it has to with that - this is the concept of affordance 😃 I understand the reasons why it got up there, but still, I think we should test another UI approach more distant from the protocol's logic and closer to people's mental models.

----
### Regarding this issue:

Yes, suppressing the tag is not the best solution, is a quick and dirty one before we can actually fix the problem (backend and UI).
 
#### "Will users realize there is something strange happening just because a green button is missing?"  
1) It is not a button. 😲 
2) It doesn't explain what is ok or what is wrong (mouseover tooltip for that was among user's feedbacks), it just shows a status and it does't help the user to fix any problem if there is one. 
3) They will not realize there is something wrong only because there's a green label missing. Agreed. But they also won't be receiving red alert messages for what they can't control. 
 
#### "Would it be better to always show red 'Not signed' if something is wrong with the signature?"
Is orange, not signed, right? 
I understand not giving any information is bad, of even worse than none. But when we got to that conclusion we were thinking about our users' transition and bringing new concepts to explain is a rather delicate thing that doesn't fit in a quick and dirty tweak. But yeah, since this is a short term solution, I'm fine with that. 

--- 

Take this scenario: I am a journalist and I want to send an email with my new article to my editor. It's nothing special, but people told me to use Pixelated for work stuff. I send this article to my editor who also has an account in Pixelated, so my email goes signed and encrypted. Cool! - That's most of our user's cases because that's what we talked in our previous meetings with them. What I'm trying to say is that is nice to have in mind that those bad cases are exceptions while we are still testing and while our users are aware this is a testing phase. 

I suggest we should work on that other UI approach (closer to sender/reciever's name/email) when we get back to really work on this issue (backend fix) and think better and together about how to signify an email signature for "normal" people.

**Comment by re-nobre** [comment]
I talked with @SweetVirginia  and we will analyse all the UX proposals in a different place and create the issues when we will work on it.

For all the UX/UI proposals and feedbacks, take a look [here] (https://docs.google.com/a/thoughtworks.com/document/d/19cBhvyfdmLY7iQ6ktRYGG6LaqYBa447Z_eg8_f1bfKw/edit?usp=sharing)

I'm closing this issue now.
